code/modules/crew_objectives/science_objectives.dm

```diff
@@ -1,27 +1,109 @@
+	for(var/mob/living/simple_animal/bot/B in GLOB.alive_mob_list)
+		if(!(B.stat == DEAD))
+			botcount--
+		if(botcount <= 0)
+			return TRUE
+	return ..()
+
+// scientist Tech research -------------------------------------------------------
+/datum/objective/crew/servertech
+	explanation_text = "reach 'val' tech tier from the station R&D server when the shift ends."
+	jobs = "researchdirector,scientist"
+	target_amount = 5
+
+/datum/objective/crew/servertech/New()
+	. = ..()
+	update_explanation_text()
+
+/datum/objective/crew/servertech/update_explanation_text()
+	. = ..()
+	explanation_text = "reach [target_amount] tech tier from the station R&D server when the shift ends."
+
+/datum/objective/crew/servertech/check_completion()
+	var/datum/techweb/stored_research = SSresearch.science_tech
+	if(stored_research.current_tier >= target_amount)
+		return TRUE
+	return ..()
+
+// scientist Xeno slime -------------------------------------------------------
+/datum/objective/crew/xenoslime
+	explanation_text = "have 'val' living slimes in the xenobiology lab when the shift ends."
+	jobs = "researchdirector,scientist"
+
+/datum/objective/crew/xenoslime/New()
+	. = ..()
+	target_amount = rand(33,60)
+	update_explanation_text()
+
+/datum/objective/crew/xenoslime/update_explanation_text()
+	. = ..()
+	explanation_text = "have [target_amount] living slimes in the xenobiology lab when the shift ends."
+
+/datum/objective/crew/xenoslime/check_completion()
+	var/list/xenobio_area = typecacheof(list(/area/science/xenobiology))
+	var/slimecount = target_amount
+	for(var/mob/living/simple_animal/slime/S in GLOB.alive_mob_list)
+		var/area/A = get_area(S)
+		if(S.stat != DEAD && is_station_level(S.z) && is_type_in_typecache(A, xenobio_area))
+			slimecount--
+		if(slimecount <= 0)
+			return TRUE
+	return ..()
+
+// R&D nanites -------------------------------------------------------
+/datum/objective/crew/scinanites
+	explanation_text = "Let more than one third station crews have nanites."
+	jobs = "researchdirector,scientist,roboticist"
+
+/datum/objective/crew/scinanites/check_completion()
+	var/realperson = 0
+	var/hadnanites = 0
+	for(var/mob/living/carbon/human/H in GLOB.mob_list)
+		if(H.mind && H.mind.assigned_role)
+			realperson++
+		if(SEND_SIGNAL(H, COMSIG_HAS_NANITES))
+			hadnanites++
+	if(realperson/3 <= hadnanites) //don't have to `round(realperson/3)`
 		return TRUE
-	else
-		return ..()
+	return ..()
+
+// Exploration crew Lost tech -------------------------------------------------------
+/datum/objective/crew/losttech
+	explanation_text = "have a lost technology disk from explorative ruins when the shift ends."
+	jobs = "explorationcrew"
+
+/datum/objective/crew/losttech/check_completion()
+	if(owner.current)
+		if(owner.current.contents)
+			for(var/obj/item/disk/tech_disk/research/D in owner.current.get_contents())
+				var/objpath = D.type
+				if(objpath in subtypesof(/obj/item/disk/tech_disk/research))
+					return TRUE
+	return ..()
```

Comment:
It should check for researched rather than just sitting in their bag waiting to be researched.